exporter/awscloudwatchlogsexporter/config.go

```diff
@@ -114,6 +124,31 @@ func isValidRetentionValue(input int64) bool {
 	return false
 }
 
+// Check if the tags input is valid
+func isValidTagsInput(input map[string]*string) string {
```

Comment:
lets make the `isValidTagsInput` function return an `error` type instead of `string`, and return `nil` in case of no error. In case we make this change, it would be more appropriated to name this function `validateTagsInput`.

You can use `fmt.Errorf` to format and generate the several error messages that you are considering. For reference: https://pkg.go.dev/fmt#Errorf

Reply:
Changes made in latest commit